src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -45,9 +45,9 @@
     "\n",
     "The flowsheet that we will be using for this module is shown below with the stream conditions. We will be processing toluene and hydrogen to produce at least 370 TPY of benzene. As shown in the flowsheet, we use a flash tank, F101, to separate out the non-condensibles, and a distillation column, D101, to further separate the benzene-toluene mixture to improve the benzene purity.  The non-condensibles separated out in F101 will be partially recycled back to M101 and the rest will be purged. We will assume ideal gas behavior for this flowsheet. The properties required for this module are defined in\n",
     "\n",
-    "- `hda_ideal_VLE.py`\n",
+    "- `ideal_VLE.py`\n",
```

Comment:
I think these need to be renamed to indicate what they are - `ideal_VLE.py` does not explain what it is.

Even though it is now in a folder for HDA properties, this is not immediately apparent from the file name (especially seeing as you haven't mentioned the folder yet) - it need to indicate the components of interest (either HDA to indicate the components of the HDA process or something similar).

Similarly the reaction package should retain the HDA - it is the reactions for the HDA process.